examples/one-page.html

```diff
@@ -9,14 +9,6 @@
   <body>
     <div id="root"></div>
 
-    <script>
-      window.userTheme = {
```

Comment:
Note: This never worked. I took it from a different branch that had integration.